octodns_powerdns/__init__.py

```diff
@@ -82,6 +72,19 @@ def __init__(
         sess.headers.update({'X-API-Key': api_key})
         self._sess = sess
 
+        self.soa_edit_api = kwargs.pop('soa_edit_api', 'DEFAULT')
+        self.mode_of_operation = kwargs.pop('mode_of_operation', 'master')
```

Comment:
Previous value was `Master` is it case insensitive?

Reply:
Yes, both of them are case insensitive (tested with v4.0 through v4.8). mode_of_operation, which is zone "kind", returns as Camel case, whereas soa_edit_api is not validated and returns as posted. Nevertheless both are working as expected despite the letter case adopted.

What stile is preferred? We can be consistent with the wording in pdns documentation, but this will make the octoDNS config ugly:
`mode_of_operation: Primary`
`soa_edit_api: DEFAULT`
 I would say to define them all lowercase to make octoDNS configuration nice without users need to search for the correct value format. We could also accept the values from config file case insensitive if we transformed the values during the validation.

Reply:
So long as it's insensitive this seems fine.